### ISSUE TITLE
No ReCaptcha V3 in Joomla 5 default contact form?

### ISSUE THREAD

**Issue by rootshosting** [SharkyKZ/joomla-recaptcha-v3-plugin#3]
Hi, I have a recently upgraded J5 website with the default Cassiopeia template, I created a default contact menu item and in the page the Recaptcha V3 plugin recently installed/configures/Google Keyed doesn't display the captcha at all. 
When I send an email it says "email sent" but then that email never arrives, tested over 10 times. 
Is there something special I should know?

**Comment by SharkyKZ** [comment]
Have you configured the Contact component to use the ReCaptcha V3 plugin?

**Comment by rootshosting** [comment]
Yes, enabled, as far as I know all is set right...

**Comment by SharkyKZ** [comment]
Inspect the form page source code and check if `media/plg_captcha_recaptcha_v3/js/main.js` script is loaded. If the plugin is really enabled but captcha validation fails, it should at least be producing an error message.

**Comment by rootshosting** [comment]
No, that line is not present in the page source, but I'm pretty sure that all is configured properly, Recaptcha V3 is set in Global Configuration, the plugin is enabled, it's also set in the contact options... and I also tested with "flexicontact" extension, all is setup there, but the recaptcha still doesn't display.

**Comment by SharkyKZ** [comment]
Honestly, no idea what could be causing such behavior. All the signs show that the plugin isn't even being used. Impossible to tell without access to the site.

**Comment by rootshosting** [comment]
Actually I've used hcaptcha now and it works, but if you want to study why
your v3 didn't work I can give you full access, no problem, contact me
privately and I give you all necessary credentials.
